chef/cookbooks/tempest/templates/default/tempest.conf.erb

```diff
@@ -406,7 +406,7 @@ api_v3_extensions = all
 #rdp_console = false
 
 # Does the test environment support resizing? (boolean value)
-resize = true
+resize=<%= @use_resize %>
```

Comment:
Just to keep the style of the config file, it'd be nice to put spaces around `=`.

+1 once done